### DISCUSSION TITLE
OPENVSP Geometry file

### DISCUSSION THREAD

**Discussion by anuj117** [OpenMDAO/Aviary#438] [General]
is there a direct function that can convert aviary's geometry file(.csv) to openvsp file(.vsp3)?

**Comment by crecine** [comment]
No, we do not. VSP files require more detailed information than Aviary does (airfoil details, fuselage cross sections, etc). In the future we might be able to add support for a conventional tube and wing, by making simplifying assumptions (cylindrical fuselage, NACA 4412, etc), but that isn't a priority at this point in time.
However, if this feature is important to you, we would encourage you to contribute and reach out if you need help.
